app/src/main/java/pm/gnosis/heimdall/ui/base/BaseFragment.kt

```diff
@@ -3,15 +3,15 @@ package pm.gnosis.heimdall.ui.base
 import android.os.Bundle
 import android.support.v4.app.Fragment
 import io.reactivex.disposables.CompositeDisposable
-import pm.gnosis.heimdall.GnosisAuthenticatorApplication
+import pm.gnosis.heimdall.HeimdallApplication
 import pm.gnosis.heimdall.di.component.ApplicationComponent
 
 abstract class BaseFragment : Fragment() {
     protected val disposables = CompositeDisposable()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        inject(GnosisAuthenticatorApplication[this.context].component)
+        inject(HeimdallApplication[this.context].component)
```

Comment:
Can we write an extension to get the app component from the context. I hate this linr xD